kolibri/logger/management/commands/generateuserdata.py

```diff
@@ -30,307 +23,61 @@ def add_arguments(self, parser):
         # Slightly different from above, in that it will only generate new facilities if there are fewer facilities
         # on the device than specified
         parser.add_argument('--facilities', type=int, default=1, dest="facilities", help="Number of facilities")
+        parser.add_argument('--no-onboarding', action='store_true', dest="no_onboarding", help="Automatically create superusers and skip onboarding")
 
```

Comment:
If you want to add a `now` argument, can add one here too? I imagine you can set the default to a callable `timezone.now`.

Reply:
What should the type of that be? String? How would a user enter a timestamp on the command line?

Reply:
We would probably have to decide on a standard strftime compatible string. Let's leave it for now, we can add it if there is a demand for it!